### DISCUSSION TITLE
DROP as nf-core modules

### DISCUSSION THREAD

**Discussion by Lucpen** [gagneurlab/drop#460] [Q&A]
Hi, 
Thanks for creating such a cool tool, it is very useful!
I saw that you guys have started to implement DROP as nf-core modules and I am really looking forwards to use it. I had two questions about it:
1) Do you guys have a time frame?
2) Do you want some help? I would be happy to collaborate in order to make the nf-core modules/subworkflows/pipeline happen :)

**Comment by c-mertes** [comment]
Dear @Lucpen, thanks for reaching out. Indeed we started to port DROP to nf-core as part of GHGA. This is still in an early stage. 

to have an efficient collaboration here, I would wait till we have our new hire I will put on this topic. So please bear with me here for probably 2 months (Point 2). From there on, I think we can have a first running beta version by the end of the year (Point 1).

**Comment by Lucpen** [comment]
Great! Thanks for your time :)
